### PR TITLE
[#2933] fixed compatibility issues with older versions

### PR DESCRIPTION
**Pull request by chuntaojun**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

fix issue #2933 

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [x] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: common/src/main/java/com/alibaba/nacos/common/utils/ByteUtils.java [hunk 1]
```diff
@@ -17,12 +17,13 @@
 package com.alibaba.nacos.common.utils;
 
 import java.nio.charset.Charset;
-import java.nio.charset.StandardCharsets;
+
```
Comment (KomachiSion) [inline common/src/main/java/com/alibaba/nacos/common/utils/ByteUtils.java:20]: Why  remove constant value？I think use constant value will be better

File: common/src/main/java/com/alibaba/nacos/common/utils/StringUtils.java [hunk 2]
```diff
@@ -41,7 +38,7 @@
     public static final String EMPTY = "";
 
     public static String newString4UTF8(byte[] bytes) {
-        return new String(bytes, Charset.forName(StandardCharsets.UTF_8.name()));
+        return new String(bytes, Charset.forName("UTF-8"));
```
Comment (KomachiSion) [inline common/src/main/java/com/alibaba/nacos/common/utils/StringUtils.java:41]: like before said, I think use constant value will be better.

File: naming/src/main/java/com/alibaba/nacos/naming/cluster/ServerListManager.java [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * The manager to globally refresh and operate server list.
+ *
+ * // TODO This object will be deleted sometime after version 1.3.0
```
Comment (KomachiSion) [inline naming/src/main/java/com/alibaba/nacos/naming/cluster/ServerListManager.java:52]: Move this TODO common to deprecated annotation may be better?

File: naming/src/main/java/com/alibaba/nacos/naming/controllers/OperatorController.java [hunk 4]
```diff
@@ -200,12 +200,37 @@ public ObjectNode getHealthyServerList(@RequestParam(required = false) boolean h
         return result;
     }
 
+    /**
+     * This interface will be removed in a future release
+     *
+     * // TODO This object will be deleted sometime after version 1.3.0
```
Comment (KomachiSion) [inline naming/src/main/java/com/alibaba/nacos/naming/controllers/OperatorController.java:206]: Move this TODO common to deprecated annotation may be better?

File: naming/src/main/java/com/alibaba/nacos/naming/controllers/OperatorController.java [hunk 5]
```diff
@@ -200,12 +200,37 @@ public ObjectNode getHealthyServerList(@RequestParam(required = false) boolean h
...
     @PutMapping("/log")
     public String setLogLevel(@RequestParam String logName, @RequestParam String logLevel) {
         Loggers.setLogLevel(logName, logLevel);
         return "ok";
     }
 
+    /**
+     * This interface will be removed in a future release
+     *
+     * // TODO This object will be deleted sometime after version 1.3.0
```
Comment (KomachiSion) [inline naming/src/main/java/com/alibaba/nacos/naming/controllers/OperatorController.java:228]: Move this TODO common to deprecated annotation may be better?

File: common/src/main/java/com/alibaba/nacos/common/utils/ByteUtils.java (hunk 1 above)
Comment (chuntaojun) [reply common/src/main/java/com/alibaba/nacos/common/utils/ByteUtils.java:20]: JDK6 does not exist with this constant

**Pull request merged**